src/bytes.rs

```diff
@@ -88,7 +98,7 @@ impl Default for Config {
             ucp: false,
             utf: false,
             utf_check: true,
-            jit: false,
+            jit: JITChoice::Never,
```

Comment:
why not auto?

Reply:
Because as far as I can tell, PCRE2 doesn't auto-enable the JIT either. You need to at least specifically call `pcre2_jit_compile_8`, so I'm carrying over that API design choice.